test/Lsp.Integration.Tests/FileOperationsTests.cs

```diff
@@ -43,14 +43,14 @@ public async Task Should_Handle_FileCreate()
         {
             var (client, server) = await Initialize(Configure, Configure);
 
-            await client.RequestWillCreateFile(
-                new WillCreateFileParams
+            await client.RequestWillCreateFiles(
```

Comment:
The plural form seems to impact the `ILanguageClient` extension methods. Is it considered a breaking change?

Reply:
I think we are fine to take the changes, but I would like us to start a breaking changes document to catalogue the breaking changes in each version.